src/get-started/java-binding.md

```diff
@@ -66,34 +66,16 @@ follow the instructions in [Contribution Guide][how-to-build].
 ### After Install
 
 - Create an environment variable `EXONUM_HOME` pointing at installation
-  location. You should also add an entry to the `PATH` variable.
+  location.
 
   ```bash
   export EXONUM_HOME=~/bin/exonum-java-0.7.0-release
-  export PATH="$PATH:$EXONUM_HOME"
```

Comment:
Shan't we move it to the manual installation section because it is not required for brew (or is it?)?

Reply:
You need to set EXONUM_HOME using Brew, but the app will be in PATH automatically.

Reply:
So maybe it's worth moving to manual section, but then I need to move setting `EXONUM_HOME` to Brew section as well

Reply:
Let's keep it as is. The only thing is that the example shows the path from the section on  manual installation, but brew will install it somewhere else. We might clarify that with a comment:

```
export EXONUM_HOME=... # Or take the path from brew output
```